### DISCUSSION TITLE
command to stop the tasmesh broker?

### DISCUSSION THREAD

**Discussion by jenniferlee1818** [arendst/Tasmota#15037] [Q&A] [answered]
I find very little documentation about tasmesh, but I'm converting most of my tasmotas to #define USE_TASMESH.

If I need to change a Tasmota setting in the web ui, I unplug my tasmesh broker and restart the device.  Does anyone know what command I could use to stop the tasmesh broker?

**Comment by barbudor** [comment, marked as answer]
There doesn't seem to be a way to stop the meshbroker. AT least not one that I can identify in the code
Could you send a `restart 1` cmnd through MQTT ?

**Reply by jenniferlee1818** [reply]
Thanks but not really. I'm trying to stop the broker for usually one minute or more, but I got it now... Rule1 OFF. Then Restart the broker.
